report_thesis/src/sections/background.tex

```diff
@@ -226,28 +226,50 @@ \subsubsection{Extra Trees Regressor (ETR)}
 However, it often achieves better generalization performance, especially in high-dimensional or noisy datasets.
 
 \subsubsection{Gradient Boosting Regression (GBR)}\label{sec:gradientboost}
-In this section we introduce \gls{gbr} primarily based on \citet{James2023AnIS}.
-\gls{gbr} is an ensemble learning method that sequentially construct multiple decision trees, each one correcting the errors of the previous one, using gradient descent and boosting techniques.
-Each tree is small, with few terminal nodes, preventing large adjustments based on a single tree's predictions.
-This ensures that each tree makes small and simple error corrections, such that each step refines the model's performance more reliably.
+The following is an overview of the \gls{gbr} algorithm based on \citet{zemel_gbr}.
```

Comment:
This cite doesn't have anything to do with us nor the equations we present in this section.

It's important that, if we cite something, that we _actually_ used that paper. The equations we have here are not found in that paper. Can't add a paper retrospectively.